community/server/src/test/java/org/neo4j/server/rest/domain/RelationshipExpanderBuilderTest.java

```diff
@@ -29,11 +29,10 @@
 import org.neo4j.graphdb.Relationship;
 import org.neo4j.graphdb.Transaction;
 import org.neo4j.kernel.impl.MyRelTypes;
-import org.neo4j.test.DatabaseRule;
-import org.neo4j.test.ImpermanentDatabaseRule;
+import org.neo4j.test.rule.DatabaseRule;
```

Comment:
Have no idea how it compiles in master, but that's where they live
https://github.com/neo4j/neo4j/blob/3.1/community/kernel/src/test/java/org/neo4j/test/rule/DatabaseRule.java#L20